src_python/habitat_sim/utils/datasets_download.py

```diff
@@ -124,6 +124,12 @@ def initialize_test_data_sources(data_path):
             "link": data_path + "datasets/rearrange_pick/replica_cad/v0",
             "version": "1.0",
         },
+        "rearrange_dataset_v1": {
+            "source": "https://dl.fbaipublicfiles.com/habitat/data/datasets/replica_cad/v1.zip",
+            "package_name": "v1.zip",
+            "link": data_path + "datasets/replica_cad",
```

Comment:
This will name the directory symlink, so may want to be specific:
```suggestion
            "link": data_path + "datasets/replica_cad/rearrange",
```
or 
```suggestion
            "link": data_path + "rearrange_pick/replica_cad/v1",
```